### PR TITLE
Add io.github.dhadder3.SkiffDesktopAdw.json

### PR DESCRIPTION
**Pull request by dhadder3**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission

### CODE REVIEW

File: io.github.SkiffDesktopAdw.json [hunk 1]
```diff
@@ -0,0 +1,63 @@
+{
+    "app-id" : "io.github.SkiffDesktopAdw",
```
Comment (hfiguiere) [inline io.github.SkiffDesktopAdw.json:2]: this app-id is invalid. github app-id contain 4 segments, the 3rd one being the associated project or username, the one the web pages are behind (if they exist)

File: io.github.SkiffDesktopAdw.json [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "45",
+    "sdk" : "org.gnome.Sdk",
+    "sdk-extensions" : [
+
+    ],
```
Comment (hfiguiere) [inline io.github.SkiffDesktopAdw.json:8]:
```suggestion
```

File: io.github.SkiffDesktopAdw.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                    "type" : "git",
+                    "url" : "https://github.com/dhadder3/skiffdesktop-adw.git",
+                    "tag" : "0.1.0",
+                    "commit" : "ae0390a7fa950309c01098aa353fe29bb4935fb6"
+                }
+            ],
+            "modules" : [
+                {
+                    "name" : "blueprint-compiler",
+                    "buildsystem" : "meson",
```
Comment (hfiguiere) [inline io.github.SkiffDesktopAdw.json:51]: you are missing the cleanup rule to delete it after build.

File: io.github.SkiffDesktopAdw.json [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    "sdk-extensions" : [
+
+    ],
+    "command" : "io.github.SkiffDesktopAdw",
+    "finish-args" : [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--share=ipc",
+        "--device=dri",
+        "--share=network"
```
Comment (hfiguiere) [inline io.github.SkiffDesktopAdw.json:15]: does this really need Network access?

Comment (dhadder3) [reply io.github.SkiffDesktopAdw.json]: Yes. The wrapper needs internet access to login to Skiff.com and access email and such. Thank you.

**Comment by hfiguiere** [comment]
bot, build io.github.dhadder3.SkiffDesktopAdw

**Comment by hfiguiere** [comment]
```
builddir/files/share/appdata/io.github.dhadder3.SkiffDesktopAdw.appdata.xml: FAILED:
• file-invalid          : <screenshot> failed to load [https://github.com/FyraLabs/skiff-desktop/raw/main/data/shot.png]
Validation of files failed
```

**Comment by hfiguiere** [comment]
It's 404 (not found) and given that it is a link to the `main` branch of some repository, it's unsurprising (it's not recommended).

**Comment by dhadder3** [comment]
Upstream repo has been updated to include preview images and the links have been updated to use githubusercontent links.

**Comment by bbhtt** [comment]
bot, build io.github.dhadder3.SkiffDesktopAdw

**Comment by bbhtt** [comment]
Please test the above build. Thanks

**Comment by dhadder3** [comment]
The build is working well. Thank you.

**Pull request closed without merging**